test/uk/gov/hmrc/excisemovementcontrolsystemapi/data/TestXml.scala

```diff
@@ -128,6 +125,113 @@ trait TestXml {
     </urn:Body>
   </urn:IE815>
 
+  lazy val IE815NoConsignor: Elem = <urn:IE815 xmlns:urn="urn:publicid:-:EC:DGTAXUD:EMCS:PHASE4:IE815:V3.01"
+                                    xmlns:urn1="urn:publicid:-:EC:DGTAXUD:EMCS:PHASE4:TMS:V3.01"
+                                    xmlns:soapenv="http://www.w3.org/2003/05/soap-envelope"
+                                    xmlns="http://www.hmrc.gov.uk/ChRIS/Service/Control"
+                                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <urn:Header>
```

Comment:
could we maybe add a method "getIE815" here that takes a consignorID in and returns the XML. This method can then be re-used by the two vals "IE815 and IE815NoConsignor" to get the xml and then it's only defined once?